pipelines/cellpainting/create_load_data.wdl

```diff
@@ -37,6 +44,8 @@ workflow create_load_data {
   call util.generate_load_data_csv as script {
     input:
       xml_file = images_directory + "/Index.idx.xml",
+      config_yaml = config_yaml,
+      plate_id = plate_id,
```

Comment:
Same question as above

Reply:
About why you specify the inputs explicitly

Reply:
In this case, `config_yaml` could have been left out.
`plate_id` is re-usable and same with `images_directory`

Reply:
Okay then that's a great reason to have `plate_id` in there, agreed